tests/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "globals": {
+    "$": true
```

Comment:
Not sure why this is needed?

Reply:
I removed `$` as a global in the top-level ESLint configuration. But tests can assume a global `$` because that's how they're importing jQuery. Makes sense?

Reply:
👍